LibsAndSamples.sln

```diff
@@ -78,12 +78,11 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MultiCloudTestApp", "tests\
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "all", "all", "{C44AADC0-1E9B-427F-AC02-642BDF8D936B}"
 	ProjectSection(SolutionItems) = preProject
-		.editorconfig = .editorconfig
+		src\.editorconfig = src\.editorconfig
```

Comment:
AFAIK .editorconfig is hierarchical, so this ensures that MSAL and MSAL.Desktop (i.e. our products) are covered. However, we want all the code to be covered. Does that produce too many errors?